Rakefile

```diff
@@ -54,6 +73,9 @@ namespace :build do
         RakeUtils.bundle_install
       end
     end
+
+    make_blockly_symlink
+    make_code_studio_symlink
```

Comment:
I understand putting this here (right after `chef-client`) but would these make more sense at the top of `build:dashboard` since, strictly speaking, they're part of the configuration for that application?